### DISCUSSION TITLE
Is there a way of viewing documentation for a specific version? E.g 2.0.0

### DISCUSSION THREAD

**Discussion by mo-joshuacolclough** [hosseinmoein/DataFrame#257] [Q&A] [answered]
This would be useful to have, as due to the compilers available to me I cannot compile any version above 2.0.0. Perhaps there is a way of doing viewing the older versions via GitHub... Thanks

**Comment by hosseinmoein** [comment, marked as answer]
The documentation is part of the release' zip file (https://github.com/hosseinmoein/DataFrame/releases). But it is not easily available on line, unfortunately.
